rqlite.yaml

```diff
@@ -0,0 +1,37 @@
+package:
+  name: rqlite
+  version: 7.14.1
+  epoch: 0
+  description: The lightweight, distributed relational database built on SQLite
+  copyright:
+    - license: MIT
+  dependencies:
+    runtime:
+      - busybox # The entrypoint script uses a shell
+environment:
+  contents:
+    packages:
+      - ca-certificates-bundle
+      - busybox
+      - build-base
+      - go
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/rqlite/rqlite
+      tag: v${{package.version}}
+      expected-commit: 3eeafc1a8bbb83acaf7fffb8b1b4ef8249102278
+      destination: rqlite
+  - runs: |
+      mkdir -p ${{targets.destdir}}/usr/bin
+      install -Dm755 docker-entrypoint.sh ${{targets.destdir}}/usr/bin
```

Comment:
The oci entrypoint should be in its own subpackage which depends on busybox.

Reply:
All set!